README.md

```diff
@@ -1,14 +1,9 @@
-# PRET
+# pret
 
-Some would say the name stands for "Pokémon Reverse Engineering Team". We consider those people "misguided".
+You can find us on:
 
-## Discord invite
-
-If we had a vanity, it would be [discord.gg/pret](https://discord.gg/d5dubZ3).
-
-## Community pages
-
-PRET is not a community.
+* Discord: [pret](https://discord.gg/d5dubZ3).
+* IRC: [libera#pret](https://web.libera.chat/?#pret)).
```

Comment:
Extra closing parenthesis here (also, do these lines need to end with a period?)